stac_fastapi/api/stac_fastapi/api/app.py

```diff
@@ -132,9 +132,7 @@ def register_landing_page(self):
             response_model_exclude_unset=False,
             response_model_exclude_none=True,
             methods=["GET"],
-            endpoint=create_async_endpoint(
-                self.client.landing_page, EmptyRequest, self.response_class
-            ),
+            endpoint=create_async_endpoint(self.client.landing_page, EmptyRequest),
```

Comment:
as we've done in #625, when we add wrap the endpoint output to a `Response` object, FastAPI will not do any serialization/validation.